src/reference-manual/statements.qmd

```diff
@@ -365,42 +366,70 @@ To access accumulated log density up to the current execution point,
 the function `target()` may be used.
 
 
-## Sampling statements {#sampling-statements.section}
+## Distribution statements {#sampling-statements.section}
 
-Stan supports writing probability statements also in sampling
-notation, such as
+Stan supports writing probability statements also using distribution
+statements, for example
 
 ```stan
 y ~ normal(mu, sigma);
+mu ~ normal(0, 10);
+sigma ~ normal(0, 1);
 ```
+The symbol $\sim$ is called tilde. Due to historical reasons, the distribution statements used to be called "sampling statements" in Stan, but that term is not recommended anymore as it is less accurate description.
```

Comment:
It's hard to read these really long lines in GitHub.  I would strongly prefer to keep our lines to 80 characters where possible.